tools/convert-help-center-docs-to-mdx

```diff
@@ -48,6 +51,37 @@ def replace_image_path(markdown_string: str) -> str:
     return result.replace('="/static/images/help-beta', '="../../../../static/images/help')
 
 
+def fix_file_imports(markdown_string: str) -> str:
+    def convert_to_pascal(text: str) -> str:
+        return to_pascal(text).replace("-", "").replace(".Md", "")
+
+    def convert_to_astro_tag(match: re.Match[str]) -> str:
+        return "<" + convert_to_pascal(match.group(1)) + " />"
+
+    def append_str_to_line(text: str, destination_str: str, n: int) -> str:
+        lines = destination_str.splitlines()
+        if 1 <= n <= len(lines):
+            lines[n - 1] += "\n" + text + "\n"
+
+        return "\n".join(lines)
+
+    RE = re.compile(r"^ {,3}\{!([^!]+)!\} *$", re.MULTILINE)
+    result = RE.sub(convert_to_astro_tag, markdown_string)
+    matches = RE.findall(markdown_string)
+
+    import_statement_set = {
+        f'import {convert_to_pascal(match)} from "./include/_{match}"' for match in matches
```

Comment:
Why the naming scheme with included filenames starting with `_`?

Reply:
Added a comment for that, posting here so it's easy to check:

```
    # We do not want Astro to render these include files as standalone
    # files, prefixing them with an underscore accomplishes that.
    # https://docs.astro.build/en/guides/routing/#excluding-pages
```